content/challenges/medium/9 - Palindrome/func.js

```diff
@@ -9,5 +9,17 @@
  * @returns {boolean}
  */
 module.exports = (str) => {
-    // Your code here
+    if (str.length == 0) {
+        return true;
+    }
+
+    for (let i = 0; i < str.length; i++) {
+        for (let k = str.length - 1; k >= 0; k--) {
+            if (str[i] === str[k]) {
+                return true;
+            } else {
+                return false;
+            }
+        }
+    }
```

Comment:
Uh oh. Looks like we have a case of bad test cases.

Add the following test case to your func.test.js file for this problem and then rework this

```
test('Expect false, is palindrome', () => {
    expect(func("abaa")).toEqual(false);
});
test('Expect false, is palindrome', () => {
    expect(func("aaba")).toEqual(false);
});
```

Right now if I test your code with this does this: 
<img width="156" alt="image" src="https://github.com/user-attachments/assets/57584119-1825-44ca-850e-5ac46090f371" />